### PR TITLE
Remove public key, closes #652

### LINKED ISSUE #652: Remove public key from account
## Which content is outdated?
accounts.adoc
transactions.adoc
Account.png

## What should be the new content?
The public key property is not part of the account anymore, hence its description is removed from accounts.adoc, and a further clarification is added to transactions.adoc. Finally, the account diagram is updated accordingly.

### PR DESCRIPTION
**Pull request by ricott1**
Remove public key from accounts, modify transaction page, modify account image

### CODE REVIEW

File: modules/ROOT/pages/transactions.adoc [hunk 1]
```diff
@@ -66,8 +66,9 @@ This implies that under normal circumstances delegates give priority to transact
 === `senderPublicKey`
 
 The public key of the account issuing the transaction.
-Note that this public key does not necessarily sign the transaction.
+Note that this public key link:1-accounts.adoc#keys[does not necessarily sign the transaction].
```
Comment (IkerAlus) [inline modules/ROOT/pages/transactions.adoc]: This link looks broken. There is no `1-accounts.adoc` file. Following the link formatting I think it should be:
```
xref:{url_accounts_keys}[`keys`]
```

**Review by IkerAlus — CHANGES_REQUESTED** [review]
The link looks broken, the rest looks good to me.

**Review by IkerAlus — DISMISSED** [review]

File: modules/ROOT/pages/transactions.adoc [hunk 2]
```diff
@@ -66,8 +66,9 @@ This implies that under normal circumstances delegates give priority to transact
 === `senderPublicKey`
 
 The public key of the account issuing the transaction.
-Note that this public key does not necessarily sign the transaction.
+Note that this public key xref:{url_accounts_keys}[does not necessarily sign the transaction].
 For simplicity, the account issuing the transaction is called the sender account in this document.
+The `senderPublicKey` is used to generate the address under which the sender account information is stored.
```
Comment (janhack) [inline modules/ROOT/pages/transactions.adoc]:
```suggestion
The `senderPublicKey` is used to compute the address under which the sender account information is stored.
```

Comment (janhack) [reply modules/ROOT/pages/transactions.adoc]: Generate sounds like a new address is created.

**Review by janhack — COMMENTED** [review]
Consider updating the formulation as suggested.

**Review by MaximeGagnebin — DISMISSED** [review]

**Review by MaximeGagnebin — APPROVED** [review]

**Review by janhack — APPROVED** [review]

**Review by IkerAlus — APPROVED** [review]

**Pull request merged**